.drone.yml

```diff
@@ -37,6 +37,45 @@ steps:
     event:
     - tag
 
+---
+kind: pipeline
+name: linux-arm64
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - sleep 20
```

Comment:
any reason the sleep here is needed?

Reply:
Copy pasta from amd64. It might not be needed but kept it to be safe.